thingsvision/thingsvision.py

```diff
@@ -36,9 +36,9 @@ def get_parsers():
     common_parser.add_argument(
         "--device",
         type=str,
-        default="cpu",
+        default="cuda",
         choices=["cpu", "cuda"],
-        help="Device to use for the extractor. (default: cpu)",
+        help="Device to use for the extractor. (default: cuda)",
```

Comment:
Ah, I was just about to comment on the default and then the next commit came in that fixed it. I was going to mention about adding in MPS options but that can be for a separate PR now, I think.

Reply:
@Alxmrphi, would you be willing to submit that PR after I've accepted the current one?